_data/simple-icons.json

```diff
@@ -15901,6 +15901,11 @@
             "hex": "333333",
             "source": "https://github.com/unocss/unocss/blob/fc2ed5bb6019b45565ff5293d4b650522f1b79b4/playground/public/icon.svg"
         },
+        {
+            "title": "unpkg",
+            "hex": "000000",
```

Comment:
I'm seeing `#242424` used as the background of their [GitHub](https://github.com/unpkg) and [Twitter](https://twitter.com/unpkg) profile pictures.

![](https://github.com/unpkg.png)